Tools/Transformation-Kompendium-nach-HTML/README.md

```diff
@@ -0,0 +1,21 @@
+# Transformation Kompendium nach HTML
+
+Diese beiden Skripte erlauben die Konvertierung der XML-Version des Kompendiums in einfach lesbare HTML-Versionen. Zur Ausführung wird eine JRE und die Saxon-Library (aktuell SaxonHE12-9J, erhältlich unter https://github.com/Saxonica/Saxon-HE/releases/download/SaxonHE12-9/SaxonHE12-9J.zip) benötigt. In den hier angegebenen Beispielen muss die Datei SaxonHE12-9J.zip in das Unterverzeichnis SaxonHE12-9J entpackt werden. Java unterstützt auch XML mit BOM in der Codierung von UTF-8.
+
+## 📄 html-komp.xsl
+
+Hiermit wird eine Version erzeugt, in der jede Praktik als eigene Tabelle dargestellt wird. Angereichert wird dies durch ein paar statistische Daten.
+
+Mögliche Nutzung mit Saxon:
+
+java -jar "SaxonHE12-9J/saxon-he-12.9.jar" "-xsl:html-komp.xsl" "-s:Grundschutz++-Kompendium.xml" "-o:GS++Komp_2025.html" -t

```

Comment:
Wenn die Zeile in einem Code-Block platziert wird, bietet GitHub einen Kopier-Button, was es manchen Personen etwas einfacher machen dürfte (ein Klick statt drei schnelle Klicks):

<img width="1319" height="136" alt="Image" src="https://github.com/user-attachments/assets/bdd70993-7e24-4cb5-9f09-c88d95963a69" />

Also so:

<img width="1088" height="101" alt="Image" src="https://github.com/user-attachments/assets/d713a502-39b8-48a1-a6db-ae3fd26ef819" />